src/org/scada_lts/web/mvc/api/PointHierarchyAPI.java

```diff
@@ -329,4 +327,71 @@ public ResponseEntity<PointHierarchyNode> getPointHierarchyWithEmptyRoot(HttpSer
             return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
         }
     }
+
+    @RequestMapping(value = "/api/pointHierarchy/moveBatch", method = RequestMethod.POST)
+    public ResponseEntity<String> moveBatch(@RequestBody PointHierarchyDTO pointHierarchyDTO, HttpServletRequest request) {
+        LOG.info("/api/pointHierarchy/moveBatch newParentXidFolder: "
+                + pointHierarchyDTO.getNewParentIdFolder() + " keys: " + pointHierarchyDTO.getXids());
+        ResponseEntity<String> result = null;
+        try {
+            User user = Common.getUser(request);
+            if (user.isAdmin()) {
+                boolean allMoved = true;
+                for (String xid : pointHierarchyDTO.getXids()) {
+                    if (xid.startsWith("DIR_")) {
+                        pointHierarchyXidService.moveFolder(xid, pointHierarchyDTO.getNewParentIdFolder());
+                    } else {
+                        boolean moved = pointHierarchyXidService.movePoint(xid, pointHierarchyDTO.getNewParentIdFolder());
+                        if (!moved) {
+                            allMoved = false;
+                        }
+                    }
+                }
+                result = new ResponseEntity<String>(String.valueOf(allMoved), HttpStatus.OK);
+            } else {
+                result = new ResponseEntity<String>(HttpStatus.UNAUTHORIZED);
+            }
+        } catch (Exception e) {
+            LOG.error(e);
+            result = new ResponseEntity<String>(HttpStatus.BAD_REQUEST);
+        }
+        return result;
+    }
+
+    @RequestMapping(value = "/api/pointHierarchy/deleteFolderMoveToRootBatch", method = RequestMethod.POST)
+    public ResponseEntity<String> deleteFolderMoveToRootBatch(
```

Comment:
Isn't the old endpoint enough?
"/api/pointHierarchy/deleteFolder/{xid_folder}"

After all, the deletion mechanism should work as before, the change only concerned moving multiple points at once?